### DISCUSSION TITLE
multiple quarkus-runs under a single Port

### DISCUSSION THREAD

**Discussion by farhangamary** [quarkusio/quarkus#29070] [Q&A]
Having multiple quarkus application each running on a different port may somehow have problems in situations like having cxf soap wsdls which display the ports or when having proxies or mappers, etc.

Does someone know a solution for running multiple quarkus applications under single port?

**Comment by gsmet** [comment]
You cannot have multiple applications sharing the same port.

What you could do however is run each application on a different port and have a frontend (either a webserver or reverse proxy) in front of your Quarkus applications. This frontend will listen on only one port and dispatch the requests to the Quarkus applications based on simple rules (could be the hostname, a specific root path...).
